docs/install/helm.md

```diff
@@ -105,23 +105,23 @@ kubectl apply -f <output_dir>/ako/crds/
 helm list -n avi-system
 
 NAME          	NAMESPACE 	REVISION	UPDATED                             	    STATUS  	CHART    	APP VERSION
-ako-1593523840	avi-system	1       	2023-04-16 13:44:31.609195757 +0000 UTC	    deployed	ako-1.10.1	1.10.1
+ako-1593523840	avi-system	1       	2023-04-16 13:44:31.609195757 +0000 UTC	    deployed	ako-1.10.3	1.10.3
 ```
 
 *Step3*
 
-Get the values.yaml for the AKO version 1.10.3 and edit the values as per the requirement.
+Get the values.yaml for the AKO version 1.11.1 and edit the values as per the requirement.
 
 ```
-helm show values oci://projects.registry.vmware.com/ako/helm-charts/ako --version 1.10.3 > values.yaml
+helm show values oci://projects.registry.vmware.com/ako/helm-charts/ako --version 1.11.1 > values.yaml
 
 ```
 *Step4*
 
 Upgrade the helm chart
 
 ```
-helm upgrade ako-1593523840  oci://projects.registry.vmware.com/ako/helm-charts/ako -f /path/to/values.yaml --version 1.10.3 --set ControllerSettings.controllerHost=<IP or Hostname> --set avicredentials.password=<username> --set avicredentials.username=<username> --namespace=avi-system
+helm upgrade ako-1593523840  oci://projects.registry.vmware.com/ako/helm-charts/ako -f /path/to/values.yaml --version 1.11.1 --set ControllerSettings.controllerHost=<IP or Hostname> --set avicredentials.password=<username> --set avicredentials.username=<username> --namespace=avi-system
```

Comment:
@DixitAakash : There is helm upgrade command present in ako_ha.md. Please update that one also.

Reply:
@akshayhavile done